lib/websocket-terminal/src/main/go/server.go

```diff
@@ -75,6 +74,37 @@ func (wp *wsPty) Stop() {
 }
 
 func ptyHandler(w http.ResponseWriter, r *http.Request) {
+	tokenParam := r.URL.Query().Get("access_token")
```

Comment:
I think if they don't use author comment in files we should not introduce such a practice too